actions.py

```diff
@@ -465,24 +580,50 @@ def submit(self, dispatcher: CollectingDispatcher,
         departure_iata = tracker.get_slot("iata_departure")
         destination_airport = tracker.get_slot("travel_destination")
         destination_iata = tracker.get_slot("iata_destination")
-
-        message = f"A one-way flight from {departure_airport} ({departure_iata}) to {destination_airport} ({destination_iata}) "\
-                  f"emits {co2_in_tons} of CO2. " \
-                  f"It would be amazing if you bought offsets for that carbon! " \
-                  f"There are some great, UN-certified projects you can pick from."
+        stopover_airport = tracker.get_slot("travel_stopover")
+        stopover_iata = tracker.get_slot("iata_stopover")
+        stored_slots = tracker.get_slot("stored_slots")
+
+        if not stopover_iata:
+            message = f"A one-way flight from {departure_airport} ({departure_iata}) to {destination_airport} ({destination_iata}) "\
+                      f"emits {co2_in_tons} of CO2. " \
+                      f"It would be amazing if you bought offsets for that carbon! " \
+                      f"There are some great, UN-certified projects you can pick from."
+        else: 
+            message = f"The trip from {departure_airport} ({departure_iata}) to {destination_airport} ({destination_iata}) "\
+                      f"via {stopover_airport} ({stopover_iata}) emits {co2_in_tons[2]+co2_in_tons[3]:.1f} of CO2. " \
```

Comment:
I would actually do the addition of the `co2_in_tons` earlier, so we don't need to change the type.